defaults/main.yml

```diff
@@ -4,7 +4,8 @@ appuio_openshift_localstorage_mountpoint_parent: /data
 appuio_openshift_localstorage_defaults: {}
 appuio_openshift_localstorage_override: {}
 
+# Per node
+appuio_openshift_localstorage_openshift_node_name: "{{ inventory_hostname }}"
```

Comment:
Assign `appuio_openshift_localstorage_node_name` and default to `inventory_hostname` to not brake backwards compatibility.